docs/core/testing/mstest-analyzers/mstest0037.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Use proper 'Assert' methods"
+description: "Learn about code analysis rule MSTEST0037: Use proper 'Assert' methods."
+ms.date: 11/17/2024
+f1_keywords:
+- MSTEST0037
+- UseProperAssertMethodsAnalyzer
+helpviewer_keywords:
+- UseProperAssertMethodsAnalyzer
+- MSTEST0037
+author: Youssef1313
+ms.author: ygerges
+---
+# MSTEST0037: Use proper 'Assert' methods
+
+| Property                            | Value                                                                  |
+|-------------------------------------|------------------------------------------------------------------------|
+| **Rule ID**                         | MSTEST0037                                                             |
+| **Title**                           | Use proper 'Assert' methods                                            |
+| **Category**                        | Usage                                                                  |
+| **Fix is breaking or non-breaking** | Non-breaking                                                           |
+| **Enabled by default**              | No                                                                     |
+| **Default severity**                | Info                                                                   |
```

Comment:
May change depending on the discussion outcome in https://github.com/microsoft/testfx/pull/4052/files#r1845291274